packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -178,23 +188,41 @@ export const prepareCosmosOrchestrationAccountKit = (
       }),
     },
     /**
-     * @param {ChainAddress} chainAddress
-     * @param {string} bondDenom e.g. 'uatom'
+     * @param {object} info
+     * @param {ChainAddress} info.chainAddress
+     * @param {string} info.bondDenom e.g. 'uatom'
+     * @param {LocalIbcAddress} info.localAddress
+     * @param {RemoteIbcAddress} info.remoteAddress
      * @param {object} io
      * @param {IcaAccount} io.account
      * @param {Remote<StorageNode>} io.storageNode
      * @param {ICQConnection | undefined} io.icqConnection
      * @param {Remote<TimerService>} io.timer
      * @returns {State}
      */
-    (chainAddress, bondDenom, io) => {
+    ({ chainAddress, bondDenom, localAddress, remoteAddress }, io) => {
       const { storageNode, ...rest } = io;
       // must be the fully synchronous maker because the kit is held in durable state
       const topicKit = makeRecorderKit(storageNode, PUBLIC_TOPICS.account[1]);
       // TODO determine what goes in vstorage https://github.com/Agoric/agoric-sdk/issues/9066
-      void E(topicKit.recorder).write('');
-
-      return { chainAddress, bondDenom, topicKit, ...rest };
+      // XXX consider parsing local/remoteAddr to portId, channelId, counterpartyPortId, counterpartyChannelId, connectionId, counterpartyConnectionId
+      // FIXME these values will not update if IcaAccount gets new values after reopening.
```

Comment:
when? as part of some issue?

Reply:
Updated the comment to mention https://github.com/Agoric/agoric-sdk/issues/9066